mongo/integration/client_side_encryption_prose_test.go

```diff
@@ -861,12 +967,10 @@ func (cpt *cseProseTest) teardown(mt *mtest.T) {
 	mt.Helper()
 
 	if cpt.cseClient != nil {
-		err := cpt.cseClient.Disconnect(mtest.Background)
-		assert.Nil(mt, err, "encrypted client Disconnect error: %v", err)
+		_ = cpt.cseClient.Disconnect(mtest.Background)
```

Comment:
Why not check for an error here?

Reply:
In one of the new tests, I share a `Client` instance between two `ClientEncryption` instances. Closing the second one yields an error because the underlying `Client` has already been closed. We generally ignore errors from `Close`/`Disconnect` functions anyway, so I changed both this and the one below to ignore for consistency.